pkg/portal/model/plan.go

```diff
@@ -0,0 +1,11 @@
+package model
+
+import (
+	"github.com/authgear/authgear-server/pkg/lib/config"
+)
+
+type Plan struct {
+	ID            string
+	Name          string
+	FeatureConfig *config.FeatureConfig
```

Comment:
This struct here is left as raw. That is, accessing the fields may trigger nil pointer exception. Do we need to use another name to remind us this?

Reply:
How about changing the pointer to struct?

Reply:
Using struct would be more problematic if we accidentally serialize it with `encoding/json`. Maybe rename it to `RawFeatureConfig`?